src/Avalonia.Controls/TextBox.cs

```diff
@@ -232,6 +235,12 @@ public int SelectionEnd
             }
         }
 
+        public int MaxLength
```

Comment:
I do not think using `int` is a good way and because we should not set any negative number to max length